src/.babelrc

```diff
@@ -1,11 +1,10 @@
 {
   "presets": [
     [
-      "env",
+      "@babel/core",
       {
         "modules": false
       }
     ]
-  ],
-  "plugins": ["external-helpers"]
```

Comment:
I _think_ you still might need this `external-helpers` plugin though. I don't think that you need to import into this file, but in _does_ need to be installed as in `devDependencies` i.e. present inside `node_modules/` for transpiling to work. I could be wrong about that.